tasks/gitrefs.js

```diff
@@ -0,0 +1,29 @@
+
+'use strict';
+
+module.exports = function(grunt) {
+
+  var templateHelper = require('./lib/templateHelper').init(grunt);
+  var _ = require('underscore');
+
+  function updateGitRefs(filepath, gitRef) {
+    var globalJson = grunt.file.readJSON(filepath);
+
+    templateHelper.walkObjects(globalJson.show, function (obj) {
+      if (_.has(obj, 'repoUrl') && _.has(obj, 'repoBranch')) {
+        obj.repoBranch = gitRef
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)